shell/platform/darwin/common/framework/Headers/common.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_COMMON_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_FRAMEWORK_HEADERS_COMMON_H_
+
+#include <objc/objc.h>
+
+/**
+ * Unique identifier for views.
+ *
+ * View IDs are generated by the embedder, and are not interpreted by the
+ * embedder API in any way.
```

Comment:
These details are very specific to the embedder/engine integration.

Is this is a header file public to end users? If yes, I would consider removing this sentence. End users likely don't know what the embedder API is. They likely don't need to know who generates these IDs either.